libbeat/monitoring/report/elasticsearch/elasticsearch.go

```diff
@@ -131,18 +131,30 @@ func makeReporter(beat beat.Info, cfg *common.Config) (report.Reporter, error) {
 
 	monitoring := monitoring.Default.GetRegistry("xpack.monitoring")
 
+	outClient := clients[0]
+	if len(clients) > 1 {
+		outClient = outputs.NewFailoverClient(clients)
+	}
```

Comment:
Why not just always initialize `outClient` as `outputs.NewFailoverClient(clients)`? Looking at the implementation of `outputs.NewFailoverClient`, it returns the first client from `clients` if `len(clients) == 1` anyway:

https://github.com/elastic/beats/blob/e1d8c15ac1cb8dc8fea8f5b23dc299e669ad4585/libbeat/outputs/failover.go#L46-L48